src/reaktoro_pse/tutorials/softening_acid_ro_example.ipynb

```diff
@@ -0,0 +1,1322 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reaktoro for modeling softening and acid addition pre-treatment for desalination systems\n",
+    "\n",
+    "\n",
+    "#### Dependencies\n",
+    "* Python - Programming language\n",
+    "* Pyomo - Python package for equation-oriented modeling\n",
+    "* IDAES - Python package extending Pyomo for flowsheet modeling\n",
+    "* cyipopt - Solver necessary for use with gray box models\n",
+    "* Reaktoro-pse - Python package for building Reaktoro gray box models\n",
+    "* watertap-solvers - for cyipopt solver wrapper"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how Reaktoro blocks can model pre-treatment and conditions inside different parts of a treatment process:\n",
```

Comment:
This example demonstrates how Reaktoro blocks can model pre-treatment and **other chemistry phenomena** in a water treatment train.

Reply:
Done